src/ngraph_skip_assert.cc

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+#include "ngraph_skip_assert.h"
+
+#include "tensorflow/core/graph/graph.h"
+#include "tensorflow/core/graph/node_builder.h"
+#include "tensorflow/core/graph/types.h"
+
+#include "ngraph_utils.h"
+
+using namespace std;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+//
+// Main entry point for skip assert.
+//
+Status SkipAssert(Graph* graph) {
+  for (auto node : graph->op_nodes()) {
+    if (node->type_string() == "Assert") {
+      NGRAPH_VLOG(4) << "Checking: " << node->name();
+      for (auto edge : node->out_edges()) {
+        if (edge->IsControlEdge()) {
+          NGRAPH_VLOG(4) << "Control edge: " << node->name();
+          graph->RemoveControlEdge(edge);
+          NGRAPH_VLOG(4) << "Control edge removed. ";
+        }
+      }
+    }
+  }
+  return Status::OK();
+}
```

Comment:
There can be no failures of TF in the lines executed above, so this function will always return `Status::OK()`

So we don't necessarily need `TF_RETURN_IF_ERROR` in this line `TF_RETURN_IF_ERROR(SkipAssert(options.graph->get()));` in `ngraph_rewrite_pass.cc`.

But in keeping with the other functions in rewrite_pass which return meaningful statuses, we could keep this function as it is now (returning Status instead of void)